src/structures/ClientUser.js

```diff
@@ -48,7 +48,7 @@ class ClientUser extends Structures.get('User') {
 
   /**
    * Edits the logged in client.
-   * @param {Object} data The new data
+   * @param {APIModifyClientData} data The new data
```

Comment:
q: This doesn't technically need to be documented since we type it's values here. CC @iCrawl @kyranet